content/english/java-tictactoe/answer-key.md

```diff
@@ -222,22 +223,56 @@ public class TicTacToe_Nuevo {
                 return "";
             }
         }
-        return "It's a TIE! Try it again";
+        return "It's a TIE! Try again";
     }
 }
 ```
 
 # activity-5: AI Computer Move
 ```java
-// reimplement getComputerMove()
-public static int getComputerMove(){
-    Random rand = new Random();
-    int pos = rand.nextInt(9) + 1;
-    while(playerMoves.contains(pos) || computerMoves.contains(pos)){
-        pos = rand.nextInt(9) + 1;
+public static int getComputerMoveAI(String[] curBoard){
+        int pos = 0;
+        int bestScore = Integer.MIN_VALUE;
+        for(int i = 1; i <= 9; i++){
```

Comment:
Looks like the CPU will always favor the top row of cells versus the bottom row (due to the ordering of which we check the board). Maybe as a bonus-bonus exercise, settle tied best scores randomly?

Reply:
I modified the rules for scores so it includes a factor of (spots available + 1), so the program will pick the move that will allow the computer to win in the fewest steps :)